conf/mesos-site.xml

```diff
@@ -63,11 +63,21 @@
   </property>
 
   <property>
+    <value>false</value>
+    <name>mesos.hdfs.preshared-hadoop-binaries</name>
```

Comment:
What does "preshared" mean? How about "preinstalled" or "predistributed"? Or reverse the polarity and call it "update-hadoop-install" (default true), or something similar?
Whatever you do, this is a complex enough property to warrant a <description> tag/comment. I'd like to see them for all properties eventually, but we can just add descriptions as we go.